alert/alert.css

```diff
@@ -0,0 +1,107 @@
+:root {
+  --info-bg: #d7f1ff;
+  --info-primary: #74c7ff;
+  --info-secondary: #98d9fe;
+  --info-text: #3bb3fc;
+
+  --success-bg: #c2f3d6;
+  --success-primary: #31d372;
+  --success-secondary: #96eab7;
+  --success-text: #22ab5d;
+
+  --danger-bg: #fee0e3;
+  --danger-primary: #fd495b;
+  --danger-secondary: #ff99a6;
+  --danger-text: #fc465b;
+
+  --warning-bg: #ffdb9a;
+  --warning-primary: #f9a40f;
+  --warning-secondary: #fed180;
+  --warning-text: #ca8501;
+}
+
+* {
+  font-size: 1rem;
+}
+
+.alert-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  gap: 1rem;
+}
+
+.alert-container div{
```

Comment:
.alert-container div is selecting all the
descendants' divs of alert-container, While I suppose these properties were meant for the main alert box. In which case, you can use .alert-container > div, It will be applied only on the direct child.

One other approach, I think will be better for this is to create a .alert sector for this and use that along with info-alert and all. It will eliminate the need for wrapping it every time with alert-container.

Reply:
I have made the changes. Thanks.